### ISSUE TITLE
Unexplained error  when more than 5 players per team

### ISSUE THREAD

**Issue by selkamand** [selkamand/valorantshiny#1]
Need to add a readable error message to underlying valorant package when number of payers on one team is set to >5. Also must add  shiny validate call to this package to properly render the error to end users

![UnexplainedError](https://user-images.githubusercontent.com/73202525/192126017-c4515763-eaeb-40d7-a185-aa7999bea299.PNG)

**Comment by selkamand** [comment]
Call that most likely needs validation and more appropriate error messages:
```
valorant::agent_roulette_customs_visualise(
        attacking_nplayers = input$in_num_attackers,
        defending_nplayers = input$in_num_defenders,
        agents_to_exclude = input$in_sel_agents_to_exclude, titlesize = 15, textsize = 6
        )
      })
```

**Comment by selkamand** [comment]
Fixed in commit [811028e](https://github.com/selkamand/valorantshiny/commit/811028e5d42482a4b8cab3333b19afafee90ca91)
